.devcontainer/docker-compose.extend.yaml

```diff
@@ -5,12 +5,14 @@ services:
     volumes:
       # Mounts the project folder to '/workspace'. While this file is in .devcontainer,
       # mounts are relative to the first file in the list, which is a level up.
-      - /tmp/freva:/tmp/freva
+      - ./freva-workdir:/tmp/freva
       - .:/workspace:cached
     env_file: .devcontainer/config.env
 
     # Overrides default command so things don't shut down after the process ends.
     command: /bin/sh -c "while sleep 1000; do :; done"
+    ports:
+      - "8000:8000"
```

Comment:
Probably I am doing something wrong, but how do you start the django-server now and access it via browser? Against my intuition these two lines prevent me from accessing localhost:8000 via webbrowser. I started django via `python manage.py runserver` inside the dev-container and could curl it from inside the container, but not from the host-machine.

Reply:
VS Code appears to have done something to handle that with automatically opened ports such that you could access them from the host. I ran into a problem with getting toil access to the url of the workflow file since whatever VS Code did it doesn't end up working for other containers in the network so I opened the port explicitly but this also meant having to tell django to listen on `0.0.0.0` and made a `make run` target for that.

Now that I have it uploading the workflow files to toil instead of accessing them via http, I could put all of this back if you wanted.

Reply:
Ok, I didn't see `make run` at first glance. Not entirely sure, but if you don't need it for now I would prefer to delete it.

FYI: In order to start django along with the frontend I execute `npm run start` which starts a second server on port 8080 and django. So if we would keep this here we would need to edit `package.json` as well.

Reply:
Isn't django the one serving the frontend though? Why would we need a second npm server?

Reply:
I should have added that the npm server is only used in the dev environment in order to allow hot-reloading. In a productive environment we collect all the js-stuff via `collectstatic` into the static folder and do not rely on a second server.